storage/src/main/java/com/brandwatch/minibcr/storage/services/MentionStoreService.java

```diff
@@ -0,0 +1,30 @@
+package com.brandwatch.minibcr.storage.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.brandwatch.minibcr.common.domain.Mention;
+import com.brandwatch.minibcr.common.repository.mention.MentionRepository;
+
+@Service
+public class MentionStoreService {
```

Comment:
Where do you use this service?

Reply:
Nowhere at the moment but I am guessing when a Mentions Consumer is created, the service will be used to save whatever mentions that are passed through kafka.